docs/DeveloperGuide.md

```diff
@@ -300,11 +549,56 @@ Use case ends.
 
 > Windows, Linux, Unix, OS-X
 
-##### Private contact detail
+## Appendix E : Product Survey
```

Comment:
bcos ~ although it may not seem so, product evaluation is one of the most important part of software engineering ~ you learn from other peoples' failures and can try to improve your app by not making the same mistakes